packages/kit/src/loader/nuxt.ts

```diff
@@ -18,43 +18,58 @@ export interface LoadNuxtOptions extends LoadNuxtConfigOptions {
   config?: LoadNuxtConfigOptions['overrides']
 }
 
-export async function loadNuxt (opts: LoadNuxtOptions): Promise<Nuxt> {
-  // Backward compatibility
-  opts.cwd = opts.cwd || opts.rootDir
-  opts.overrides = opts.overrides || opts.config || {}
-
-  // Apply dev as config override
-  opts.overrides.dev = !!opts.dev
-
+export async function importNuxtPackage<T = any> (rootDir = process.cwd()): Promise<{ version: 2 | 3, package: T }> {
```

Comment:
We could also support `{ version: '2' | '3' }` option for both perf and making explicit nuxt version resolution